fill/fill_station.cc

```diff
@@ -35,7 +37,9 @@ using grpc::ServerContext;
 using grpc::ServerWriter;
 using grpc::Status;
 
-QD qd; 
+/* Actuators */
+QD qd; // Note: QD must go first as it initializes wiringPi
```

Comment:
Better to call setup in each constructor and then order doesn't matter I'd think. I confirmed calling setup multiple times should have no negative impact